src/components/Info/Form.js

```diff
@@ -81,11 +85,11 @@ export default ({ onSubmit, city }) => {
           id="email"
           type="email"
           // TODO: re-add + sign as a valid pattern before the @ symbol
-          pattern="[a-z0-9._%-]+@[a-z0-9.-]+\.[a-z]{2,4}$"
+          pattern="[a-z0-9._+%-]+@[a-z0-9.-]+\.[a-z]{2,4}$"
```

Comment:
This looks good to me. Can you remove the `TODO` only line 83 since the plus sign has been re-added to the pattern?

Reply:
Sure